frontend/packages/helm-plugin/src/actions/useRollbackAction.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Action } from '@console/dynamic-plugin-sdk';
+import { HelmRelease } from '../types/helm-types';
+import { fetchHelmReleaseHistory } from '../utils/helm-utils';
+import { getHelmRollbackAction } from './creators';
+import { HelmActionsScope } from './types';
+
+const useRollbackAction = (scope: HelmActionsScope) => {
+  const { t } = useTranslation();
+  const action = React.useRef<Action>(getHelmRollbackAction(scope, t));
+  const [rollBackAction, setRollBackAction] = React.useState<Action[]>([]);
+
+  React.useEffect(() => {
+    let ignore = false;
+
+    const getRollbackAction = async () => {
+      let releases: HelmRelease[];
+      try {
+        releases = await fetchHelmReleaseHistory(scope.releaseName, scope.namespace);
```

Comment:
You don't actually need to fetch history. Helm release has data about revision. If revision is `1` then you don't need to show a rollback option.